src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  --big-height: 70px;
+  --small-height: 42px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.input {
+  background: #fff;
+  height: 100%;
+  width: 100%;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  padding-left: 33px;
+  line-height: 19px;
+  outline: none;
+  font-family: inherit;
+  font-weight: 300;
+  color: #3d4e61;
+  font-size: 14px;
+}
+
+.inputBig {
```

Comment:
Try to rework it according to checklist. Just create one class for input and then specification in input--small 
![Screenshot 2023-03-15 at 17 28 48](https://user-images.githubusercontent.com/6526149/225359408-e96059dd-a093-47e8-a72a-a5635ddfcf09.png)